projects.yaml

```diff
@@ -53,6 +53,9 @@ module:
     hic_lpc11u35: &module_hic_lpc11u35
         - records/rtos/rtos-cm0.yaml
         - records/hic_hal/lpc11u35.yaml
+    hic_lpc11u35_bm: &module_hic_lpc11u35_bm
```

Comment:
As these might not be related to GCC port, please send t hem separately  or after GCC support lands on master.